packages/tocco-ui/src/Panel/Panel.js

```diff
@@ -65,7 +70,11 @@ Panel.propTypes = {
   /**
    * Boolean to control if body can be opened or closed. Default value is 'true'.
    */
-  isToggleable: PropTypes.bool
+  isToggleable: PropTypes.bool,
+  /**
+   * Function is triggered by componentDidUpdate or by toggleOpenState.
```

Comment:
I still wouldn't understand what this sentence means :) No one who uses this component cares how `componentDidUpdate` is implemented, and what is toggleOpenState? Why not simply write "Function that is called when Panel is opened or closed"?